src/components/ComposedModal/ComposedModal.js

```diff
@@ -20,6 +20,7 @@ ComposedModal.propTypes = {
   modalHeaderProps: PropTypes.object,
   modalTrigger: PropTypes.func,
   onCloseModal: PropTypes.func,
+  size: PropTypes.string,
```

Comment:
You can update this prop type to reflect the available options for size props.

```
PropTypes.oneOf([
    'xs',
    'sm',
    'md',
    'lg',
  ])
```

Reply:
done